website/docs/d/iam_trusted_profiles.html.markdown

```diff
@@ -0,0 +1,76 @@
+---
+layout: "ibm"
+page_title: "IBM : ibm_iam_trusted_profiles"
+description: |-
+  Get information about iam_trusted_profiles
+subcategory: "IAM Identity Services"
+---
+
+# ibm_iam_trusted_profiles
+
+Provides a read-only data source for iam_trusted_profiles. You can then reference the fields of the data source in other resources within the same configuration using interpolation syntax.
+
+## Example Usage
+
+```hcl
+data "ibm_iam_trusted_profiles" "iam_trusted_profiles" {
+	profile_id = "profile_id"
+}
+```
+
+## Argument Reference
+
+Review the argument reference that you can specify for your data source.
+
+* `profile_id` - (Required, Forces new resource, String) ID of the trusted profile to get.
+
+## Attribute Reference
```

Comment:
R in Reference to be as reference.  General comment in all the website/docs file